app/assets/javascripts/uploads.coffee

```diff
@@ -0,0 +1,24 @@
+$(document).ready ->
+  reIntializeMasonry = (reinitial) ->
+    setTimeout (->
+      $container = $('#container')
+      if reinitial
+        $container.masonry('destroy')
+      # initialize
+      $container.masonry
+        itemSelector: '.item'
+      ), 100
+
+  reIntializeMasonry()
+
+  $('#fileupload').fileupload
+    dataType: 'json'
+    done: (e, data) ->
+      file = data.result
+      $("<div class='item col-md-3'><img class=\"img-responsive\" src='#{file}' /></div>").prependTo $('#container')
+      reIntializeMasonry(1)
+
+  $(window).scroll ->
+    url = $('.pagination .next').find('a').attr('href')
```

Comment:
you can use `$('.pagination .next a')` instead of `$('.pagination .next').find('a')`,
use `prop` instead of `attr`